spec/observables/dom/ajax-spec.ts

```diff
@@ -948,85 +948,6 @@ describe('ajax', () => {
     delete root.XMLHttpRequest.prototype.ontimeout;
   });
 
-  it('should work fine when XMLHttpRequest onprogress property is monkey patched', function () {
```

Comment:
These tests are no longer relevant, as they were only for when we were using the property method versions of event handlers like `xhr.onprogress`. Now we're using `xhr.addEventListener('progress', ...)`, so it's a non-issue. Since Zone.js is patching `EventTarget`, this should "just work". Also, hopefully in the not-so-distant future, Zone.js goes away.